src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -27,11 +27,12 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
           {props.error ? <Text style={styles.errorText}>{props.error}</Text> : null}
         </View>
         {!props.isOtherProfile && (
-          <Button
-            styles={styles.button}
-            textStyling={styles.textStyling}
-            onPress={props.handleSave}>
-            {props.isLoadingUpdateBio ? <ActivityIndicator size="small" color="#0000ff" /> : 'Save'}
+          <Button style={styles.button} textStyling={styles.textStyling} onPress={props.handleSave}>
+            {props.isLoadingUpdateBio ? (
+              <ActivityIndicator size="small" color={COLORS.darkBlue} />
+            ) : (
+              'Save'
+            )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ActivityIndicator` has been changed from a hardcoded value to a constant from the theme file. This is a good practice as it improves maintainability and consistency across the application.
```diff
- <ActivityIndicator size="small" color="#0000ff" />
+ <ActivityIndicator size="small" color={COLORS.darkBlue} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->